### PR TITLE
Buttwoo

### PR DESCRIPTION
**Pull request by arj03**
Spec: https://github.com/ssbc/ssb-buttwoo-spec
ssb uri2 PR: https://github.com/ssbc/ssb-uri2/pull/7

### CODE REVIEW

**Comment by staltz** [comment]
How final do you think the design is? Are you confident in inserting this?

**Comment by mycognosist** [comment]
I've taken a quick read of the spec. Will reply with comments tomorrow.

**Comment by arj03** [comment]
@staltz I feel happy about the spec in its current form. Have a good confidance that it is works well enough in [practice](https://github.com/ssbc/ssb-ebt/pull/67#issuecomment-1128947585). Maybe @mycognosist will change my mind :)

@mycognosist ohh what a pleasant surprise, I look forward to hearing what input you have.

**Comment by mixmix** [comment]
this change in this repo feels like to me. 
the one that's harder to change is ssb-bfe where we have a programmatic aspect and assign feeds slots.
that I'm more hesitant about adding to because finite slots. but I think we should be less precious and more permissive. I want to enable @arj03 so I'm a hearty approve 👍

**Review by mixmix — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -34,9 +34,11 @@ The specification in this document is compatible with `ssb-uri` while adding sup
   - `ssb:message/sha256/<MSGID>`
   - `ssb:message/bendybutt-v1/<MSGID>`
   - `ssb:message/gabbygrove-v1/<MSGID>`
+  - `ssb:message/buttwoo-v1/<MSGID>`
```
Comment (staltz) [inline readme.md:37]: So is it `buttwoo` or `buttwoo-v1`?

Comment (staltz) [reply readme.md:37]: I don't mind either way

**Comment by mycognosist** [comment]
I posted a small request for clarification of the spec: https://github.com/ssbc/ssb-buttwoo-spec/issues/7

As far as this PR goes, it looks good to me. I think I lean towards the `buttwoo-v1` naming.

**Review by mycognosist — APPROVED** [review]
🟢

**Review by staltz — APPROVED** [review]
Just want to clarify that I don't have any blockers to merge. :)

**Pull request merged**